### PR TITLE
Update ru.json with new strings from en.json

### PR DESCRIPTION
**Pull request by hide-me**
### What does it do?

Added new lines with translation from en.json

### Why is it needed?

To complete full translation of new lines into Russian.

### How to test it?

No need to test at all.

### CODE REVIEW

File: packages/core/admin/admin/src/translations/ru.json [hunk 1]
```diff
@@ -559,6 +563,31 @@
...
+  "components.Blocks.modifiers.italic": "Курсив",
+  "components.Blocks.modifiers.strikethrough": "Зачёркнутый",
+  "components.Blocks.modifiers.underline": "Подчёркнутый",
+  "components.Blocks.popover.cancel": "Отмена",
+  "components.Blocks.popover.delete": "Удалить",
+  "components.Blocks.popover.edit": "Редактировать",
+  "components.Blocks.popover.link": "Ссылка",
+  "components.Blocks.popover.save": "Сохранить",
+  "components.Blocks.popover.text": "Текст",
+  "components.Blocks.popover.text.placeholder": "Введите текст ссылки",
```
Comment (madhurisandbhor) [inline packages/core/admin/admin/src/translations/ru.json:590]: Please update to the latest placeholder text "Enter link text" , Текст ссылки

**Review by madhurisandbhor — COMMENTED** [review]
All changes looks good except 1, please update, thanks

File: packages/core/admin/admin/src/translations/ru.json (hunk 1 above)
Comment (hide-me) [reply packages/core/admin/admin/src/translations/ru.json]: Done!
Thank you for take a look. Now it seems ready?

**Review by Imran-imtiaz48 — COMMENTED** [review]
### Review and Improvement

The provided code consists of translation keys and their corresponding values in Russian. There is redundancy in the code where certain keys and values are repeated, leading to unnecessary duplication. This can make the file harder to maintain and increase the risk of inconsistencies.

### Improved Configuration

To improve the configuration, we should eliminate the duplicated keys and ensure each key is unique within the file. Here's the streamlined version:

```json
{
  "components.Blocks.popover.link": "Ссылка",
  "components.Blocks.popover.save": "Сохранить",
  "components.Blocks.popover.text": "Текст",
  "components.Blocks.popover.text.placeholder": "Введите текст ссылки",
  "components.ErrorBoundary.title": "Что-то пошло не так...",
  "components.FilterOptions.FILTER_TYPES.$contains": "содержит",
  "components.FilterOptions.FILTER_TYPES.$containsi": "содержит (без учета регистра)"
}
```

This revised configuration removes the duplicate entries, reducing the file size and improving readability. Additionally, maintaining a unique set of keys ensures that any future updates or translations are easier to manage and less prone to errors.

**Comment by hide-me** [comment]
Wow... after 8 months...

**Comment by alexandrebodin** [comment]
> [!IMPORTANT] 
> With the release of Strapi 5, we have lifted the code freeze. To ensure we can effectively review and assist with contributions, we are now only accepting PRs targeting version 5 (the develop branch).<br/> 
> As your PR targets version 4, it has been automatically closed. However, we would greatly appreciate it if you could update your contribution to align with version 5 and resubmit!

**Comment by hide-me** [comment]
ok

**Pull request closed without merging**